core/sqf/sql/scripts/sqnodestatus

```diff
@@ -26,12 +26,11 @@
 # Implements:
 #   ssh -M "ConnectTimeout 1" -o "ConnectionAttempts 3" n028 hostinfo 2>&1
 #
-# Uses:
-#   export MY_NODES=' -w n028 -w n044 -w n029 -w n045 -w n030 -w n031 -w n032 '
+# Uses the following enviroment variables:
 #   export SQ_MON_SSH_OPTIONS=' -o "ConnectTimeout 1" -o "ConnectionAttempts 3" '
 #
 
-my $node_context=lc(readpipe("echo \$NODE_LIST"));
+my $node_context=readpipe("trafconf -name");
```

Comment:
The upper lower case is addressed in other code. Specifically, in the monitor code. The trafconf utility is new and was created to address the need of obtaining the current hostname (node name) membership in a Trafodion cluster instance. I will add more details to its purpose and usage in the JIRA: TRAFODION-2001. A brief summary is as follows:

- Node membership is initially derived from 'sqconfig' and used to populate the Trafodion Configuration database. After this, node membership is through the 'shell' node add/delete commands.
- The Trafodion Configuration database is the configuration of record once the instance is started.
- The 'trafconf' utility reads the contents of the Trafodion Configuration database to display the current node membership of the cluster.
- The 'trafconf' utility us used by scripts and by the Trafodion user .bashrc to populate environment variables with the current node membership. Specifically, the MY_NODES environment variable.